resources/language/resource.language.es_es/strings.po

```diff
@@ -126,11 +128,11 @@ msgstr "Biblioteca"
 
 msgctxt "#30026"
 msgid "Enable custom library folder"
-msgstr "Habilitar carpeta personalizada de la biblioteca"
+msgstr "Habilitar carpeta para biblioteca personalizada"
```

Comment:
PARTIALLY AGREE. Use instead: "Habilitar carpeta para **la** biblioteca personalizada". Notice the word 'la' in bold.